### PR TITLE
Add unitary fund as affiliation supporting qutip

### PR DESCRIPTION
**Pull request by nathanshammah**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,17 +21,17 @@
     var txt;
     if (lng.indexOf("zh") !== -1)
     {
-        txt = "感谢您使用QuTiP。如果您在论文中使用了QuTiP，请您记得引用本篇论文。"; 
+        txt = "感谢您使用QuTiP。如果您在论文中使用了QuTiP，请您记得引用本篇论文。";
```
Comment (BoxiLi) [inline index.html:24]: I didn't know that there is Chinese :)

**Comment by Ericgig** [comment]
Seems good, but I don't know enough html to make sure it will look as expected. But I don't see how it could break the site, so merging.

**Comment by Ericgig** [comment]
Look good, links are working.

**Comment by nathanshammah** [comment]
@Ericgig you are right. I generally look at stuff locally with my browser. This helps only partially due to bootstrap. For example the logo looks huge in plain html (`index.html`) from browser. I generally merge and then check online. Luckily it worked OK. http://qutip.org/. 

If someone comes up with a simple way to check locally, that would help me. I always do this a bit in the dark when updating the website. I did not have the time to look for bootstrap editors and so, but if someone knosw this stuff and can write a ` how-to.md`, it would be nice. cc @tehruhn.

**Pull request merged**